README.md

```diff
@@ -1 +1,236 @@
+ 
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+- **Ability to see the featured doctors on the homepage**
+- **The ability to reserve doctors based on selecting the date and doctor name**
+- **The ability to add doctors to the database**
+- **The ability to delete doctors**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+Live demo for the website [Coming Soon]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎬 Kanban Board <a id="kanban-board"></a>
+
+Link to the [Kanban Board](https://github.com/tajulafreen/Final-capstone-Backend/projects/1)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need to:
+
+- Install Ruby
+- Install Rails
+- Install Posgresql
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``` 
+  git clone https://github.com/tajulafreen/Final-capstone-Backend.git 
+```
+
+### Install
+
+Install this project with these commands:
+
+```
+  cd hotel-hub-backend
+  bundle install
+  rails db:create
+  rails db:migrate
+  rails db:seed
+  ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Database Setup.
+
+Before creating database, you have to configure the master key for your project. This can be done in two ways.
+
+1. **Using our own master key that we have provided**
+cd into the project folder and in your terminal run
+
+``` 
+rails rake:secret
+
+```
+- This command will generate a master.key file and inside that file replace the key with the one shown below.
+
+**[4099d6598a781b627e9488716c68382f]**
+
+- Proceed to creating db.
+
+2. **Generating your own master Key**
+
+- Remove config/master.key and config/credentials.yml.enc if they exist.
+- Run in the terminal: EDITOR=code rails credentials:edit 
+- Close the editor that opens.
+- This command will create a new master.key and credentials.yml.enc if they do not exist.
+
+- After this proceed to creating db with:
+```
+  rails db:create
+  rails db:migrate
+  rails db:seed
+  ```
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+ 
+To run the project execute the following command:
+
+``` 
+ rails s
+```
+
+or
+
+```
+./bin/dev
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+###  Tests <a id="run-tests"></a>
+
+To run the tests for models and requests, execute the following command:
+
+```
+
+rspec ./spec/api/v1
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍